localization/i18n/es/OrcaSlicer_es.po

```diff
@@ -6243,8 +6247,8 @@ msgstr ""
 "El archivo %s ya existe\n"
 "¿Desea reemplazarlo?"
 
-msgid "Confirm Save As"
-msgstr "Salvar Como"
+msgid "Comfirm Save As"
```

Comment:
Its Confirm.

Reply:
Translated as "Confirmar Guardar como" and also all "Salvar" as "Guardar"